.github/workflows/deploy.yml

```diff
@@ -0,0 +1,39 @@
+name: Deploy to GitHub Pages
+
+on:
+  # Trigger the workflow every time you push to the `main` branch
+  # Using a different branch name? Replace `main` with your branch’s name
+  push:
+    branches: [feat/website]
```

Comment:
Does this work now in this branch?

Reply:
Yes, but since it's a fork, it pushes in our repo, when merged, it should be changed to main. You can see the link here for instance https://pheno-agency.github.io/proxy-memoize/

Reply:
Hm, okay, let's change it to `main` in this repo in this PR and we will see.